docs/transactions/index.md

```diff
@@ -28,8 +28,9 @@ Make API calls that include the transaction identifier to include them in the tr
 
 Supported operations include:
 
-- Read and write documents.
+- Read and write documents to collections.
 - Get the number of documents in a collection.
+- Get the number of key-value pairs in a collection.
```

Comment:
I think this item is unnecessary, as it duplicates the previous item but for a KV collection. Maybe "Get and update the groups of a KV collection" or something similar could address the KV Group feature.